lerna.json

```diff
@@ -2,5 +2,10 @@
   "packages": ["packages/*"],
   "version": "1.1.1",
   "npmClient": "yarn",
-  "useWorkspaces": true
+  "useWorkspaces": true,
+  "command": {
+    "publish": {
+      "message": "chore(release): Publish %s"
```

Comment:
Note that the default release bot only support `fix`, `feat` and `BREAKING CHANGE`. I'm not sure if this would be included in bot's log.

Reply:
This could be excluded in the bot log, because it is only used for the release commit message. e.g. `chore(release): Publish v1.2.0`

Reply:
Thanks for reminders that bot only detect the `fix`, `feat` and `BREAKING CHANGE`. 😃

Reply:
I keep in mind while writing the commit messages.